app/javascript/App/components/NewsfeedPage/NewsList.js

```diff
@@ -20,6 +20,7 @@ class NewsList extends Component {
       this.setState({ initialRender: false })
     }, 60000)
     this.mountOnScrollHandler()
+    document.querySelector('#newsfeed').style.height = `${window.outerHeight}px`
```

Comment:
@cryptoB0T Shouldn't this technically be `window.innerHeight` minus the height of the topnav minus the height of the searchbar? Otherwise the scrollbar behaves oddly.

Reply:
@hanchang nice catch, thanks!